tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -534,6 +534,13 @@ def from_tensors(tensors):
     >>> list(dataset.as_numpy_iterator())
     [(array([1, 2, 3], dtype=int32), b'A')]
 
+    >>> # `from_tensors` does not change the shape of input tensor.
+    >>> # use `from_tensor_slices` to slice the input tensor.
+    >>> example = tf.constant([1,2,3])
+    >>> dataset = tf.data.Dataset.from_tensors(example).repeat(1000)
+    >>> np.array(list(dataset1.as_numpy_iterator())).shape
```

Comment:
This is a complicated way to print the shape. Can we simplify it to `dataset.element_spec`, or change the example so that we can print all elements of the dataset with `list(dataset.as_numpy_iterator())`?

Reply:
done.